fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -3842,14 +3843,68 @@ public ParseNode visitOdbcFunctionCallExpression(StarRocksParser.OdbcFunctionCal
                     FunctionSet.ADDDATE,
                     FunctionSet.DATE_ADD, FunctionSet.DATE_SUB,
                     FunctionSet.SUBDATE,
-                    FunctionSet.DAYS_SUB,
-                    FunctionSet.TIME_SLICE);
+                    FunctionSet.DAYS_SUB);
+
+    private static List<Expr> getArgumentsForTimeSlice(Expr time, Expr value, String ident, String boundary) {
+        List<Expr> exprs = Lists.newLinkedList();
+        exprs.add(time);
+        exprs.add(value);
+        exprs.add(new StringLiteral(ident));
+        exprs.add(new StringLiteral(boundary));
+
+        return exprs;
+    }
 
     @Override
     public ParseNode visitSimpleFunctionCall(StarRocksParser.SimpleFunctionCallContext context) {
 
         String functionName = getQualifiedName(context.qualifiedName()).toString().toLowerCase();
 
+        FunctionName fnName = FunctionName.createFnName(functionName);
```

Comment:
Both IntervalLiteral and UnitBoundary are Expr, can they be directly used as parameters of FunctionCallExpr without special treatment?

Reply:
We must disassemble IntervalLiteral to get two parts. so can't use IntervalLiteral directly.